content/blog/2018-06-06-2018-unconf-security-recap.md

```diff
@@ -0,0 +1,53 @@
+
+Unconf projects x: 
+
+---
+slug: "unconf_recap_x"
+title: "Unconf18 projects x: middlechild, defender, ropsec, keybase"
+date: 2018-06-xx
+authors:
+  - name: Karthik Ram
+categories: blog
+topicid: 
+tags:
+- community
+- ropensci
+- meetings
+- unconf
+---
+
```

Comment:
Please add this where appropriate:

"In the spirit of exploration and experimentation at rOpenSci unconferences, these projects are not necessarily finished products or in scope for rOpenSci packages."